README.md

```diff
@@ -30,6 +30,8 @@ jobs:
           cwd: "subDirectory (optional)"
           # Optional. Compare only this benchmark target
           benchName: "example-bench-target"
+          # Optional. Disable the default features of a crate
+          noDefaultFeatures: true
```

Comment:
Hmmm using a double negation is a bit misleading. I would prefer a configuration similar to that in Cargo.toml:

```yaml
defaultFeatures: false
```

By default it would be `true`. In any case, this requires checking for its existence:
 - If it's set, take its value, if not, default to `true`.

I think I still need some inputs from more people, though.

Reply:
Fair point, the double negation might be problematic. I'll switch to the `defaultFeatures` flag, that's closer to the way it is defined in a `Cargo.toml`